### PR TITLE
Android course in English

### PR DESCRIPTION
**Pull request by raj007-star**
## What does this PR do?
Add resource(s) - Learn Android Dev in English

### Why is this valuable (or not)?
This course is valuable for people who are beginner in Android dev and wants to learn Kotlin in Android dev. 

### How do we know it's really free?
it is available on Youtube platform and youtube is free.

### For book lists, is it a book? For course lists, is it a course? etc.
It is a Android dev course in English language.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -216,6 +216,7 @@
 * [Firebase in a Weekend: Android](https://www.udacity.com/course/firebase-in-a-weekend-by-google-android--ud0352) (Udacity)
 * [Gradle for Android and Java](https://www.udacity.com/course/gradle-for-android-and-java--ud867) (Udacity)
 * [Jetpack Compose](https://www.youtube.com/playlist?list=PLQkwcJG4YTCSpJ2NLhDTHhi6XBNfk9WiC) - Philipp Lackner
+* [Kotlin & Android 12 Tutorial | Learn How to Build an Android App 📱 9+ h FREE Development Masterclass](https://www.youtube.com/watch?v=HwoxgUPabMk) - tutorialsEU
```
Comment (sohansai) [inline courses/free-courses-en.md:219]:
```suggestion
* [Kotlin & Android 12 Tutorial - Learn How to Build an Android App 9+ h FREE Development Masterclass](https://www.youtube.com/watch?v=HwoxgUPabMk) - tutorialsEU
```

Comment (raj007-star) [reply courses/free-courses-en.md:219]: I am using the full link only I dont know bro

Comment (raj007-star) [reply courses/free-courses-en.md]: Is it all right

**Comment by raj007-star** [comment]
> For youtube, we prefer "playlist" links

Okay do I need to change it again?

**Comment by sohansai** [comment]
> > For youtube, we prefer "playlist" links
> 
> Okay do I need to change it again?

Apologies for any confusion; it would be greatly appreciated if you could provide a playlist link instead of a single video link, as it aligns better with our preferences.

**Comment by raj007-star** [comment]
The playlist contain almost every thing a app developer need from android sql to deploying your apps. Please check this it will be really helpful for beginner. I did this course and really helped me clearing the basics.

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -198,6 +198,7 @@
 * [Advanced Android App Development](https://www.udacity.com/course/advanced-android-app-development--ud855) (Udacity)
 * [Advanced Android with Kotlin](https://www.udacity.com/course/advanced-android-with-kotlin--ud940) (Udacity)
 * [Android App Development for Beginners Playlist](https://www.youtube.com/playlist?list=PL6gx4Cwl9DGBsvRxJJOzG4r4k_zLKrnxl) - Bucky Roberts (thenewboston)
+* [Android app Development Tutorial for Beginners (2020)](https://youtube.com/playlist?list=PLwhVruPHD9rz2MwQuYSenQ0WK1IGP5rYM&si=e83lmCD1uY1fLVaT) - tutorialsEU
```
Comment (eshellman) [inline courses/free-courses-en.md:201]:
```suggestion
* [Android app Development Tutorial for Beginners (2020)](https://youtube.com/playlist?list=PLwhVruPHD9rz2MwQuYSenQ0WK1IGP5rYM) - tutorialsEU
```
remove tracking parameter!

**Pull request closed without merging**